src/lib/one-graph/client.js

```diff
@@ -0,0 +1,835 @@
+const os = require('os')
+
+const dotProp = require('dot-prop')
+const { buildClientSchema, parse } = require('graphql')
+const fetch = require('node-fetch')
+
+const { NETLIFYDEVLOG, NETLIFYDEVWARN, chalk, log } = require('../../utils')
+
+const {
+  defaultExampleOperationsDoc,
+  extractFunctionsFromOperationDoc,
+  generateFunctionsFile,
+  generateHandler,
+  readGraphQLOperationsSourceFile,
+  writeGraphQLOperationsSourceFile,
+  writeGraphQLSchemaFile,
+} = require('./netlify-graph')
+
+const ONEDASH_APP_ID = '0b066ba6-ed39-4db8-a497-ba0be34d5b2a'
+
+const httpOkLow = 200
+const httpOkHigh = 299
+
+const basicPost = async (url, options) => {
```

Comment:
please add a small jsdoc comment describing the functions use case and what it does.